From 72cfae1e1f90eb24adf7203815dca19126fa0422 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 8 Jun 2018 10:36:30 +0800
Subject: [PATCH] arm64: dts: rockchip: enable ddrfreq for rk3326 linux evb
 board

As the other boards had support the dmc function, maybe the owner
was missing this dts configure.

In order to be better the power consumption and performance, this patch
enables the dmc and dfi to support the ddrfreq,

Change-Id: I2623943085d098a65ba6e0ad7102f671f0cedd3e
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
index 3ca25895939f..be94b2bd16dd 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-linux-lp3-v10.dts
@@ -323,6 +323,15 @@
 	status = "disabled";
 };
 
+&dfi {
+	status = "okay";
+};
+
+&dmc {
+	center-supply = <&vdd_logic>;
+	status = "okay";
+};
+
 &emmc {
 	bus-width = <8>;
 	cap-mmc-highspeed;
-- 
2.35.3

